compio-net/tests/udp.rs

```diff
@@ -64,57 +64,3 @@ async fn send_to() {
         active_addr
     );
 }
-
-#[compio_macros::test]
-async fn send_msg_with_ipv6_ecn() {
```

Comment:
Why remove this test?

Reply:
It's now redundant with test added in [compio-quic::socket::tests::ecn_v6](https://github.com/AsakuraMizu/compio/blob/e99bc8d23baa8c17fb9360fdfe124faa71652111/compio-quic/src/socket.rs#L699), which actually does the same thing and is better implemented. The test code here uses too many `#[cfg]` and `unsafe` which doesn't look good.